### ISSUE TITLE
improve bonus round point visuals

### ISSUE THREAD

**Issue by matthewcarleton** [rhdemo/2021-frontend-css-html#24]
@evanshortiss do we have a point value assigned to each click in the bonus round?

**Comment by evanshortiss** [comment]
@matthewcarleton not a formally agreed upon value, but @utherp0 is calculating it based on `$SHOTS_FIRED * $MULTIPLIER`. The multiplier can be configured to whatever we like pretty easily since it's a variable injected at runtime. I belive it's 5 right now.

**Comment by utherp0** [comment]
Yup, defaults to 5 but can be overridden by ENV in the bonusevent function....

**Comment by utherp0** [comment]
We also need to finalise the sink-scores; I set some random ones up, again injected via ENV into the attackevent, but they were just picked by me on the spot
